### PR TITLE
add debug logging statement to print certificate path

### PR DESCRIPTION
**Pull request by kdaily**
Print out the certificate path when in debug mode.

### CODE REVIEW

File: botocore/httpsession.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def get_cert_path(verify):
     if verify is not True:
         return verify
 
+    cert_path = where()
```
Comment (vz10) [inline botocore/httpsession.py:48]: Probably makes sense to return 'cert_path' instead of calling where() twice.

**Review by vz10 — COMMENTED** [review]
Looks good. One small NIT comment.

File: botocore/httpsession.py (hunk 1 above)
Comment (kdaily) [reply botocore/httpsession.py:48]: 🤦🏻 forgot to push that, corrected. Thanks!

**Review by vz10 — APPROVED** [review]
🚢

File: botocore/httpsession.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def get_cert_path(verify):
     if verify is not True:
         return verify
 
-    return where()
+    cert_path = where()
+    logger.debug("Certificate path: {0}".format(cert_path))
```
Comment (nateprewitt) [inline botocore/httpsession.py:49]: Just as a stylistic note, we don't need to enumerate formatting args anymore and can just use `{}` or `{cert_path}`. The numbering requirement was dropped in Python 2.6.

**Review by nateprewitt — APPROVED** [review]

**Pull request merged**